Tasks/validate.js

```diff
@@ -1,24 +1,14 @@
+'use strict';
 // Validate person name
 
-const isValidate = (T) => {
-  if (!T) return false;
-  if (T === '') return false;
-  if (typeof T !== 'string') return false;
-  if (T.length === 0) return false;
-  if (!T.includes(' ')) return false;
-  {
-    for (C of T) {
-      if (C === ' ') continue;
-      if (
-        C.toLowerCase().charCodeAt(0) >= 97 &&
-        C.toLowerCase().charCodeAt(0) <= 122
-      ) {
-      } else {
-        return false;
-      }
-    }
-    return true;
+const isValidate = (personName) => {
+  if (personName.trim().length === 0) return false;
+  if (!personName.includes(' ')) return false;
+  for (const char of personName) {
+    if (char === ' ') continue;
+    if (!char.toLowerCase().match(/[a-z]/)) return false;
```

Comment:
Better use `charAt` or `charCodeAt` and compare with `a` and `z` or with ASCII codes of letters